client/src/main/java/com/flipkart/flux/client/config/FluxClientConfiguration.java

```diff
@@ -62,4 +66,11 @@ public long getConnectionTimeout() {
     public void setConnectionTimeout(long connectionTimeout) {
         this.connectionTimeout = connectionTimeout;
     }
+    public String getClientElbId() {
+        return clientElbId;
+    }
+    public void setClientElbId(String clientElbId) {
```

Comment:
Also this field should be immutable.

Reply:
Removed setter for ClientElbId, immutable now.

Reply:
Removed setter for ClientELbId, it's immutable now.